constants/psyTokenPrice.ts

```diff
@@ -0,0 +1 @@
+export const psyDAOTokenPrice = 0.1;
```

Comment:
can price be updated? If yes, why not put it to ENV and export env into const?

Reply:
From meetings that we had they told us token sale price will stay 0.1 so that's why I used a const

Reply:
alrighty